scrapy/core/scraper.py

```diff
@@ -26,7 +26,7 @@ class Slot(object):
 
     MIN_RESPONSE_SIZE = 1024
 
-    def __init__(self, max_active_size=5000000):
+    def __init__(self, max_active_size):
```

Comment:
Removing the default will break existent code, we should keep it for backwards compatibility.